javascript/player.js

```diff
@@ -1,28 +1,120 @@
 import { SVG_COLLECTION } from './svgCollection.js';
+import { createCrossDirectionArray } from './utils.js';
 
 export default class Player {
-  constructor(board) {
-    this.clickedBoard = Array.from(Array(9), () => Array(9).fill(''));
+  constructor(board, flagNumber) {
+    this.clickedBoard = Array.from(Array(9), () => Array(9).fill(null));
```

Comment:
```suggestion
    this.clickedBoard = Array.from(Array(9), () => Array(9).fill(null));
```


Array.from(Array(9), () => Array(9).fill(null));
이 부분이 반복적으로 있는 것 같아서, 함수 하나 만들어두고, 계속 함수 호출하는건 어때요?? 
이 부분은 제안이므로, 당장 수정해야할 필요는 없습니다!

Reply:
의견 수용! 적용 완료